package.json

```diff
@@ -13,13 +13,12 @@
 		"images",
 		"easel"
 	],
-	"homepage":"https://github.com/wdamien/node-easel",
-	"repository":"git://github.com/wdamien/node-easel",
```

Comment:
These urls are published on the npm registry: https://www.npmjs.com/package/node-easel

...which made it confusing that the main repo is actually under the CreateJS github organisation